staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go

```diff
@@ -130,7 +130,7 @@ func NewCmdCreateSecretGeneric(f cmdutil.Factory, ioStreams genericclioptions.IO
 	o := NewSecretOptions(ioStreams)
 
 	cmd := &cobra.Command{
-		Use:                   "generic NAME [--type=string] [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run=server|client|none]",
+		Use:                   "secret generic NAME [--type=string] [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run=server|client|none]",
```

Comment:
I don't think that we should add "secret" word at here.

Reply:
I notice that generic is the subcommand of secret. And it is correct when i use
```shell
kubectl create secret generic -h
...
...
Usage:
  kubectl create secret generic NAME [--type=string] [--from-file=[key=]source]
[--from-literal=key1=value1] [--dry-run=server|client|none] [options]
```

But the doc generated by codes is incorrect.
See issue: https://github.com/kubernetes/website/issues/38626.


Do I miss something?

Reply:
Yeah this breaks the help message for me

```
kubernetes on 🌱fix/command [$!] via 🐹 v1.19.4 on ☁️  (us-west-2) 
❯ ./_output/bin/kubectl create secret -h
Create a secret using specified subcommand.

Available Commands:
  secret        Create a secret for use with a Docker registry
  secret        Create a TLS secret
  secret        Create a secret from a local file, directory, or literal value

Usage:
  kubectl create secret [flags] [options]

Use "kubectl <command> --help" for more information about a given command.
Use "kubectl options" for a list of global command-line options (applies to all commands).

kubernetes on 🌱fix/command [$!] via 🐹 v1.19.4 on ☁️  (us-west-2) 
❯ ./_output/bin/kubectl create secret generic -h
Create a secret using specified subcommand.

Available Commands:
  secret        Create a secret for use with a Docker registry
  secret        Create a TLS secret
  secret        Create a secret from a local file, directory, or literal value

Usage:
  kubectl create secret [flags] [options]

Use "kubectl <command> --help" for more information about a given command.
Use "kubectl options" for a list of global command-line options (applies to all commands).

kubernetes on 🌱fix/command [$!] via 🐹 v1.19.4 on ☁️  (us-west-2) 
❯ 
```

I think that this issue is not on the kubectl side but on the code that generates the reference cheat sheet side.